frontend/src/components/Admin/TeamEvent/TeamEvents.tsx

```diff
@@ -82,9 +82,13 @@ const TeamEvents: React.FC = () => {
           <h2>View All Team Events</h2>
           <TeamEventsDisplay isLoading={isLoading} teamEvents={teamEvents} />
         </div>
-        <div className={styles.resetButtonContainer}>
+        <div className={styles.buttonContainer}>
+          <Button>
+            <Link href="/admin/team-events/dashboard">View Team Events Dashboard</Link>
+          </Button>
           <ClearTeamEventsModal setTeamEvents={setTeamEvents} />
         </div>
+        <div></div>
```

Comment:
Should we remove this empty `<div>` tag?

Reply:
yup, definitely just forgot to remove it